config/routes.rb

```diff
@@ -3,11 +3,13 @@
     resources :users
     post 'auth/register', to: 'users#register'
     post 'auth/login', to: 'users#login'
-    get 'test', to: 'users#test'
-    post 'message/:id', to: 'messages#send_message'
-    get 'chatroom', to: 'chatrooms#index'
-    get 'chatroom/:id', to: 'chatrooms#show' 
-    get 'connect', to: 'users#connect'
+    get 'connection', to: 'chatrooms#get_connect'
+    post 'connection', to: 'users#connect'
+    delete 'connection', to: 'users#disconnect'
+    post 'chatrooms/:id/messages', to: 'messages#send_message'
+    get 'chatrooms', to: 'chatrooms#index'
+    get 'chatrooms/:id', to: 'chatrooms#all_messages'
+    get 'chatrooms/:id/messages', to: 'chatrooms#show' 
```

Comment:
This is basically okay, but it usually is recommended to use the ressources syntax provided by the routes domain language, because it makes the file a little bit more compact